src/screens/AnonymousTicketPurchase/components/Actions.tsx

```diff
@@ -27,6 +27,10 @@ const Actions = ({
       params: {
         afterLogin: {
           routeName: 'TabNavigator',
+          routeParams:
+            callerRoute === 'ConsequencesFromTicketPurchase'
+              ? {screen: 'Ticketing'}
+              : {},
```

Comment:
Generally the coupling between Actions-component and the components which uses it feels little iffy. They are not decoupled, but a pretty tight coupling as the Actions-component is aware of its users and has business logic dependant on it. If possible, and not to much work, I would suggest a refactor of this so the Actions-component is more decoupled from its usages.